src/senaite/core/profiles/default/rolemap.xml

```diff
@@ -3,6 +3,21 @@
 
   <permissions>
 
+    <!-- SAMPLE -->
+    <permission name="senaite.core: Transition: Multi Results" acquire="False">
+      <role name="Analyst"/>
+      <role name="LabClerk"/>
+      <role name="LabManager"/>
+      <role name="Manager"/>
+      <role name="Preserver"/>
+      <role name="Publisher"/>
+      <role name="RegulatoryInspector"/>
+      <role name="Sampler"/>
+      <role name="SamplingCoordinator"/>
+      <role name="Verifier"/>
+    </permission>
```

Comment:
I think the permission should resemble to the sum of `senaite.core: Edit Field Results`, `senaite.core: Edit Results` and `senaite.core: Field: Edit Analysis Result` permissions. Thus, I suggest:

```xml
<permission name="senaite.core: Transition: Multi Results" acquire="False">
    <role name="Analyst"/>
    <role name="LabManager"/>
    <role name="Manager"/>
    <role name="Sampler"/>
</permission>
```
IMO, `LabClerk`, `Publisher`, `RegulatoryInspector` and `Verifier` should never have this permission granted

Reply:
The problem that arises here is that anyone who can add a sample will get an insufficient privileges when the immediate results entry option is activated, e.g. Client or LabClerk. Thus, we need to keep it as it is + add Client and Owner

Reply:
noted :+1: